### PR TITLE
avoid merging configuration

### LINKED ISSUE #42: Confusing options merging
If for example I have a config.edn file with `:paths ["src" "test"]`, and I pass options from CLI 
`:paths ["src"]` because I only want to see src now, the config from cli is not respected and still use `:paths ["src" "test"]`.

Other options get applied correctly instead, so it's quite confusing until you see how the code behaves, where collections are always merged.

IMHO it would be simpler and clearer to just ignore the default config file if you pass arguments from the CLI, since I would guess if you run it locally your really want to have certain options passed in.

### PR DESCRIPTION
**Pull request by AndreaCrotti**
The configuration passed in from the CLI should simply override whatever was set in carve/config.edn instead of trying to merge it partially.
Closes #42

### CODE REVIEW

**Comment by borkdude** [comment]
Tests are failing. Please also update the README.

File: src/carve/main.clj [hunk 1]
```diff
@@ -73,13 +68,14 @@
                  (edn/read-string (slurp config-file)))]
     (if (and (not flag) (not config))
       (binding [*err* *out*]
-        (println "No config found in .carve/config.edn.\nSee https://github.com/borkdude/carve#usage on how to use carve.")
+        (println "No config found in .carve/config.edn and no --opts passed in.\nSee https://github.com/borkdude/carve#usage on how to use carve.")
```
Comment (borkdude) [inline src/carve/main.clj:71]: Can you undo this change?

File: src/carve/main.clj [hunk 2]
```diff
@@ -73,13 +68,14 @@
...
         1)
       (do (when (and (not (= "--opts" flag)) (not config))
             (throw (ex-info (str "Unrecognized option: " flag) {:flag flag})))
           (let [opts (load-opts config opts)
                 format (-> opts :report :format)
                 report (impl/run! opts)]
+
```
Comment (borkdude) [inline src/carve/main.clj:78]: This change doesn't seem to be significant.

File: src/carve/main.clj (hunk 1 above)
Comment (AndreaCrotti) [reply src/carve/main.clj:71]: Done
